### PR TITLE
Add filter-By-Services functionality

### PR DESCRIPTION
**Pull request by ibramoha**
Hi @ibramoha I left some comments for you and try every time before push your code to run the lint, please.

### CODE REVIEW

File: frontend/src/Components/Organisation/index.js [hunk 1]
```diff
@@ -62,13 +62,15 @@ export default class Organisations extends Component {
     editIdx: -1,
     category: getSelectedCategory(this.props.match),
     day: null,
+    service: null,
   };
   componentWillReceiveProps(newProps) {
     console.log(this.state.day);
```
Comment (DwinaTech) [inline frontend/src/Components/Organisation/index.js]: remove this console.log

File: frontend/src/Components/Organisation/index.js [hunk 2]
```diff
@@ -102,6 +107,36 @@ export default class Organisations extends Component {
...
+  };
+
+  handleServiceChange = service => {
+    this.setState(
+      {
+        service,
+      },
+      this.filterByService(service),
+    );
+    console.log('helo');
```
Comment (DwinaTech) [inline frontend/src/Components/Organisation/index.js]: ditto

File: frontend/src/Components/Organisation/index.js [hunk 3]
```diff
@@ -122,11 +157,10 @@ export default class Organisations extends Component {
     });
   };
   render() {
-    const { editIdx } = this.state;
-    const { category } = this.state;
-    const { day } = this.state;
+    const { editIdx, category, day, service } = this.state;
     console.info(category);
     console.info(day);
+    console.info(service);
```
Comment (DwinaTech) [inline frontend/src/Components/Organisation/index.js]: ditto

File: frontend/src/Components/Organisation/index.js [hunk 4]
```diff
@@ -80,6 +82,8 @@ export default class Organisations extends Component {
     }
     const { category } = this.state;
     const filteredOrg = originalOrganisations[category];
+    console.log(filteredOrg);
```
[the remark this answers is not in the document]
Comment (ibramoha) [reply frontend/src/Components/Organisation/index.js]: ok

File: frontend/src/Components/Organisation/index.js (hunk 1 above)
Comment (ibramoha) [reply frontend/src/Components/Organisation/index.js]: ditto

**Comment by kkarimi** [comment]
@ibramoha is this ready now?

**Comment by kkarimi** [comment]
@ibramoha Update this branch

**Comment by ibramoha** [comment]
yes @kkarimi .

**Pull request merged**